Marlin/src/lcd/menu/menu_advanced.cpp

```diff
@@ -67,7 +67,7 @@ void menu_backlash();
     static xyze_uint8_t driverPercent;
     LOOP_LOGICAL_AXES(i) driverPercent[i] = stepper_dac.get_current_percent((AxisEnum)i);
     START_MENU();
-    BACK_ITEM(MSG_ADVANCED_SETTINGS);
+    TERN_(HAS_BACK_ITEM, BACK_ITEM(MSG_ADVANCED_SETTINGS));
```

Comment:
You already know what I'm going to say about this change to every single usage of `BACK_ITEM`, yes?

Reply:
The number of files touched by this PR down is cut down to 4 by changing the macro defines instead.